rambo/Vagrantfile

```diff
@@ -101,6 +101,9 @@ custom_code_dirs = [
 Vagrant.configure("2") do |config|
   if get_env_var_rb('SYNC') != 'disabled'
     config.vm.synced_folder '.', "/vagrant", type: get_env_var_rb('SYNC')
+    if get_env_var_rb("SECRETS_PATH")
+      config.vm.synced_folder File.join(get_env_var_rb("CWD"), get_env_var_rb("SECRETS_PATH")), "/vagrant/secrets", type: get_env_var_rb('SYNC')
```

Comment:
On a technical note, I think `get_env_var_rb("SECRETS_PATH")), "/vagrant/secrets"`  could just be added to the list of synced custom dirs above in line 91. That would keep this block simpler.